src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -5,6 +5,28 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+namespace Microsoft.Win32.SafeHandles
+{
+    [System.Security.SecurityCriticalAttribute]
+    public abstract partial class SafeHandleMinusOneIsInvalid : System.Runtime.InteropServices.SafeHandle
+    {
+        protected SafeHandleMinusOneIsInvalid(bool ownsHandle) : base(System.IntPtr.Zero, ownsHandle) { }
+        public override bool IsInvalid { [System.Security.SecurityCriticalAttribute]get { return default(bool); } }
+    }
+    [System.Security.SecurityCriticalAttribute]
+    public abstract partial class SafeHandleZeroOrMinusOneIsInvalid : System.Runtime.InteropServices.SafeHandle
+    {
+        protected SafeHandleZeroOrMinusOneIsInvalid(bool ownsHandle) : base(System.IntPtr.Zero, ownsHandle) { }
+        public override bool IsInvalid { [System.Security.SecurityCriticalAttribute]get { return default(bool); } }
+    }
```

Comment:
I was initially trying to clean these up but ran into a build issue when referencing the latest System.Runtime. There are a number of assemblies affected by this and @weshaggard mentioned that referencing the new System.Runtime would cause it to not work for older framework versions. So I'm guessing that all of those would have to be retargeted for netstandard1.7 (from netstandard1.3). I'm not sure when would be an appropriate time to do that. Thoughts?